% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
\documentclass[
]{book}
\usepackage{xcolor}
\usepackage{amsmath,amssymb}
\setcounter{secnumdepth}{5}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math} % this also loads fontspec
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
\usepackage{lmodern}
\ifPDFTeX\else
  % xetex/luatex font selection
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{longtable,booktabs,array}
\usepackage{calc} % for calculating minipage widths
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}
\usepackage{graphicx}
\makeatletter
\newsavebox\pandoc@box
\newcommand*\pandocbounded[1]{% scales image to fit in text height/width
  \sbox\pandoc@box{#1}%
  \Gscale@div\@tempa{\textheight}{\dimexpr\ht\pandoc@box+\dp\pandoc@box\relax}%
  \Gscale@div\@tempb{\linewidth}{\wd\pandoc@box}%
  \ifdim\@tempb\p@<\@tempa\p@\let\@tempa\@tempb\fi% select the smaller of both
  \ifdim\@tempa\p@<\p@\scalebox{\@tempa}{\usebox\pandoc@box}%
  \else\usebox{\pandoc@box}%
  \fi%
}
% Set default figure placement to htbp
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\usepackage[]{natbib}
\bibliographystyle{plainnat}
\usepackage{booktabs}

\usepackage{color}
\usepackage{framed}
\setlength{\fboxsep}{.8em}

% These colours were manually entered, they shouldn't matter unless you want pdf output

\newenvironment{redbox}{
  \definecolor{shadecolor}{RGB}{243, 154, 157}
  \color{white}
  \begin{shaded}}
 {\end{shaded}}

\newenvironment{bluebox}{
  \definecolor{shadecolor}{RGB}{172, 210, 237}
  \color{white}
  \begin{shaded}}
 {\end{shaded}}

\newenvironment{greenbox}{
  \definecolor{shadecolor}{RGB}{141, 181, 128}
  \color{white}
  \begin{shaded}}
 {\end{shaded}}
\usepackage{bookmark}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\urlstyle{same}
\hypersetup{
  pdftitle={Microbial Genomics 2025},
  pdfauthor={Faculty: Andrew McArthur, Fiona Brinkman, Gary van Domselaar, Jared Simpson, Jimmy Liu, Rob Beiko, William Hsiao, Charlie Barclay},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}

\title{Microbial Genomics 2025}
\author{Faculty: Andrew McArthur, Fiona Brinkman, Gary van Domselaar, Jared Simpson, Jimmy Liu, Rob Beiko, William Hsiao, Charlie Barclay}
\date{November 20-22, 2025}

\begin{document}
\maketitle

{
\setcounter{tocdepth}{1}
\tableofcontents
}
\part{Introduction}\label{part-introduction}

\chapter{Workshop Info}\label{workshop-info}

Welcome to the 2025 Microbial Genomics Canadian Bioinformatics Workshop webpage!

\section{Pre-work}\label{pre-work}

You can find your pre work here:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Please read \href{./content-files/2023\%20CARD\%20DB\%20publication.pdf}{\textbf{2023 CARD DB publication}}.
\item
  Please read \href{./content-files/2023\%20CARDShark\%20publication.pdf}{\textbf{2023 CARDShark publication}}.
\item
  Please read \href{./content-files/2025\%20Bait\%20Capture\%20VERSION\%202\%20publication\%20(CARPDM).pdf}{\textbf{2025 Bait Capture VERSION 2 publication}}.
\item
  Please read \href{./content-files/2025\%20CARD\%20k-mers\%20pre-print.pdf}{\textbf{2025 CARD k-mers pre-print}}.
\item
  Please read \href{./content-files/2025\%20CZ-ID\%20publication.pdf}{\textbf{2025 CZ-ID publication}}.
\end{enumerate}

\section{Class Photo}\label{class-photo}

Coming soon!

\section{Schedule}\label{schedule}

\chapter{Meet Your Faculty}\label{meet-your-faculty}

\subsubsection{Andrew McArthur}\label{andrew-mcarthur}

\begin{quote}
Professor
Michael G. DeGroote Institute for Infectious Disease Research
Department of Biochemistry and Biomedical Sciences
McMaster University
Hamilton, ON, Canada

\begin{center}\rule{0.5\linewidth}{0.5pt}\end{center}
\end{quote}

Dr.~McArthur is a Professor and David Braley Chair in Computational Biology at McMaster University. Dr.~McArthur has had a career in the United States and Canada, including NIH-funded positions at the Marine Biological Laboratory and Brown University, where he led the genome assembly of the diarrheal pathogen Giardia intestinalis, plus 10 years of experience in the private sector. Dr.~McArthur's research team focuses on building tools, databases, and algorithms for the genomic surveillance of infectious pathogens. He and his team developed the Comprehensive Antibiotic Resistance Database (card.mcmaster.ca) and the SARS-CoV-2 Illumina GeNome Assembly Line software platform

\subsubsection{Fiona Brinkman}\label{fiona-brinkman}

\begin{quote}
Distinguished Professor, FRSC, Department of Molecular Biology and Biochemistry;
Associate Member, School of Computing Science and Faculty of Health Sciences
Simon Fraser University
Burnaby, BC, Canada

--- \href{mailto:brinkman@sfu.ca}{\nolinkurl{brinkman@sfu.ca}}, \url{https://www.brinkmanlab.ca/}
\end{quote}

Dr.~Brinkman is developing bioinformatic resources to better track infectious diseases using genomic data, and improve prediction of new vaccine/drug targets. Her primary aim is to develop more sustainable, integrated approaches for infectious disease control, however she is also applying her methods to aid allergy, child health, and environmental research.

\subsubsection{Gary Van Domselaar}\label{gary-van-domselaar}

\begin{quote}
Chief, Bioinformatics
National Microbiology Laboratory
Public Health Agency of Canada
Winnipeg, MB, Canada

--- \href{mailto:gary.vandomselaar@phac-aspc.gc.ca}{\nolinkurl{gary.vandomselaar@phac-aspc.gc.ca}}
\end{quote}

Dr.~Gary Van Domselaar, PhD (University of Alberta, 2003) is the Chief of the Bioinformatics Section at the National Microbiology Laboratory in Winnipeg Canada and Associate Professor in the Department of Medical Microbiology and Infectious Diseases at the University of Manitoba. Dr.~Van Domselaar's lab develops bioinformatics methods and pipelines to understand, track, and control circulating infectious diseases in Canada and globally. His research and development

\subsubsection{Jared Simpson}\label{jared-simpson}

\begin{quote}
Principal Investigator, Ontario Institute for Cancer Research
Assistant Prof.~Department of Computer Science, University of Toronto
Toronto, ON, Canada

--- \url{https://simpsonlab.github.io/}
\end{quote}

Dr.~Simpson develops algorithms and software for the analysis of high-throughput sequencing data. He is interested in de novo assembly and the detection of sequence variation in individuals, cancers and populations, with a focus on long read sequencing technologies. Dr.~Simpson developed the ABYSS, SGA and nanopolish software packages.

\subsubsection{Jimmy Liu}\label{jimmy-liu}

\begin{quote}
PhD Candidate, Simon Fraser University
Burnaby, BC
Canada

--- \href{mailto:ccl40@sfu.ca}{\nolinkurl{ccl40@sfu.ca}}
\end{quote}

Jimmy Liu is a PhD candidate trained in bioinformatics and biochemistry. He conducted his PhD research at Simon Fraser University focusing on the development of novel computational methods to enhance the utility of WGS data for enteric outbreak investigations and longitudinal surveillance. He also has years of experience working with nanopore long-read technologies in the context of veterinary diagnostics, such as expediting diagnostic turnaround times through real-time nanopore data analysis.

\subsubsection{Rob Beiko}\label{rob-beiko}

\begin{quote}
Professor, Faculty of Computer Science
Dalhousie University
Halifax, NS, Canada

--- \href{mailto:rbeiko@dal.ca}{\nolinkurl{rbeiko@dal.ca}}
\end{quote}

Rob Beiko is a professor in bioinformatics whose research encompasses microbial evolution and gene transfer, comparative genomics, and microbial community analysis. He is the lead of the ARETE project, which is developing a software pipeline for annotation of AMR and mobile genetic elements, phylogenomic analysis, and inference of recombination and transfer in large sets of pathogen genomes. He has also developed and contributed to software tools including STAMP for statistical analysis of metagenome profiles, rSPR for efficient inference of gene transmission, and PICRUSt for the prediction of metagenome composition based on marker-gene surveys.

\subsubsection{William Hsiao}\label{william-hsiao}

\begin{quote}
Associate Professor, Faculty of Health Sciences
Simon Fraser University
Vancouver, BC, Canada
--- \href{mailto:wwhsiao@sfu.ca}{\nolinkurl{wwhsiao@sfu.ca}}, www.cidgoh.ca
\end{quote}

William Hsiao is a public health infectious disease researcher with a background in microbial genomics and bioinformatics. He is an associate professor in the Faculty of Health Sciences at Simon Fraser University and an affiliated researcher at BCCDC Public Health Laboratory and at Canada's Michael Smith Genome Sciences Centre. Will leads an interdisciplinary group of researchers interested in solving practical public health and animal health problems through a One Health lens at the Centre for Infectious Disease Genomics and One Health.

\subsubsection{Charlie Barclay}\label{charlie-barclay}

\begin{quote}
MSc Graduate Student Researcher, Department of Physics and Astronomy
University of British Columbia
Vancouver, BC, Canada
--- \href{mailto:cbarcl01@mail.ubc.ca}{\nolinkurl{cbarcl01@mail.ubc.ca}}
\end{quote}

William Hsiao is a public health infectious disease researcher with a Charlie is an ontology curator at the Centre for Infectious Disease Genomics and One Health (CIDGOH), focusing on data standards for contextual data of genomic epidemiology, including wastewater surveillance. With five years of experience in data management, specializing in biodiversity and genomics data, she also actively contributes to the Public Health Alliance for Genomic Epidemiology (PHA4GE) and the Global Alliance for Genomics and Health (GA4GH).

\chapter{Data and Compute Setup}\label{data-and-compute-setup}

\subsubsection{Course data downloads}\label{course-data-downloads}

Coming soon!

\subsubsection{Compute setup}\label{compute-setup}

Coming soon!

\part{Modules}\label{part-modules}

\chapter{Module 1 Microbial Genome Sequencing}\label{module-1-microbial-genome-sequencing}

\section{Lecture}\label{lecture}

\section{Lab}\label{lab}

\chapter{Module 2 Genome Assembly and Annotation}\label{module-2-genome-assembly-and-annotation}

\section{Lecture}\label{lecture-1}

\section{Lab}\label{lab-1}

\chapter{Module 3 Mobile Genetic Elements}\label{module-3-mobile-genetic-elements}

\section{Lecture}\label{lecture-2}

\section{Lab}\label{lab-2}

\chapter{Module 4 Antimicrobial Resistant Gene (AMR) Analysis}\label{module-4-antimicrobial-resistant-gene-amr-analysis}

\section{Lecture}\label{lecture-3}

\section{Lab}\label{lab-3}

\chapter{Module 5 Pan Genomics}\label{module-5-pan-genomics}

\section{Lecture}\label{lecture-4}

\section{Lab}\label{lab-4}

\chapter{Module 6 Phylogenetics}\label{module-6-phylogenetics}

\section{Lecture}\label{lecture-5}

\section{Lab}\label{lab-5}

\bibliography{book.bib,packages.bib}

\end{document}
